app/models/alert_error.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: alert_errors
+#
+#  alert_generation_run_id :bigint(8)
+#  alert_type_id           :bigint(8)
+#  asof_date               :date
+#  created_at              :datetime         not null
+#  id                      :bigint(8)        not null, primary key
+#  information             :text
+#  school_id               :bigint(8)
+#  updated_at              :datetime         not null
+#
+# Indexes
+#
+#  index_alert_errors_on_alert_generation_run_id  (alert_generation_run_id)
+#  index_alert_errors_on_alert_type_id            (alert_type_id)
+#  index_alert_errors_on_school_id                (school_id)
+#
+
+class AlertError < ApplicationRecord
+  belongs_to :school
```

Comment:
No need for school on here as it can be inferred from the run? Same with alerts although there might be more to unpick there....